backend/packages/Upgrade/src/init/seed/EnableMetricFiltering.ts

```diff
@@ -1,10 +1,11 @@
 import { SettingService } from '../../api/services/SettingService';
 import { Setting } from '../../api/models/Setting';
 import Container from 'typedi';
+import { UpgradeLogger } from '../../lib/logger/UpgradeLogger';
 
 export async function enableMetricFiltering(): Promise<Setting> {
   const settingService: SettingService = Container.get(SettingService);
-  const settingDoc = await settingService.getClientCheck();
+  const settingDoc = await settingService.getClientCheck(new UpgradeLogger());
```

Comment:
We don't need logging in seeding. This was suppose to be used for fake data.

Reply:
ok